### DISCUSSION TITLE
Custom icons in PrimeOne 2.2.0

### DISCUSSION THREAD

**Discussion by karol-ona** [primefaces/community#1420] [PrimeIcons] [answered]
Is it possible to change the icons in Figma UI Kit to custom ones and still have them in the proper colors in the components? How should I do it?
![Zrzut ekranu 2024-03-12 o 13 50 06](https://github.com/primefaces/community/assets/163139729/5c81cf38-4f5b-4c2a-8419-a512b6769ce7)
The top arrow is the original icon, the bottom one I've replaced in the icon component with a custom design. All the settings I can see seem to be the same. Would be grateful for the information on how to do it, I am still learning to use the whole kit most efficiently.

**Comment by onursenture** [comment, marked as answer]
Hi, There are two important things you should do to maintain its color while swapping the icons:

The custom icon should have the same layer structure as PrimeOne icons: a single vector layer named "Vector". Note that it's case-sensitive.

<img width="425" alt="image" src="https://github.com/primefaces/community/assets/244957/bb059775-5a22-422b-a10c-b14548f0ef76">

The vector shape should be outlined.

<img width="437" alt="image" src="https://github.com/primefaces/community/assets/244957/43765171-7bd2-461a-baba-19fe3e06e6b1">
